src/documents/FrameworkConcepts.md

```diff
@@ -8,31 +8,56 @@ The Fluid Infusion Framework is built on a number of important concepts. This pa
+If everything that is important to you about the object has been successfully cloned, then it is a suitable model object.
 
-Note that this does not prohibit functions, DOM nodes, etc. from appearing in a model, just so long as the meaning of the object tree is not destroyed through the operation of `fluid.copy()`. The point here is that Infusion doesn't impose a formal contract on model objects: as long as `fluid.copy()` succeeds, you're good. For reference, the current implementation of `fluid.copy()` simply forwards the operation to [`jQuery.extend(true, ...)`](http://docs.jquery.com/Utilities/jQuery.extend). In practice, your model objects will often be JSON-compatible data suitable for sharing between your JavaScript code and other services on the Web.
+We recommend that all objects stored in the `model` area of a component conform to an even stricter definition - that they are equivalent to their form as serialised as JSON. This implies that they
+consist of only the following types: `Array`, `Object`, `String`, `Number` and `Boolean`. In addition, the following values for `Number` should not be used - `Infinity`, `-Infinity` and `NaN`. The value
+`undefined` also can not be safely stored within a model.
 
 An analogy from another environment, Java, is a [POJO](http://en.wikipedia.org/wiki/POJO) or "Plain Old Java Object". The J could just as easily be substituted for "JavaScript" in this case.
 
 ## EL Paths
 
-In some parts of the framework, we refer to "EL expressions". This is a somewhat historical phrase that perhaps sounds like it says more than it is trying to. All we mean by EL expressions are dot-separated paths built of names—for example if you had defined an object `zar` with a member `boo` which has a member `baz`, you could access the nested JavaScript property by writing the expression `zar.boo.baz`. If it were held in a string value, this would become an EL expression—that is, the string `"zar.boo.baz"` is an EL expression which designates the same piece of data we just referenced. The framework includes machinery for interpreting such expressions held in strings rather than at the JavaScript language level. This is useful because it abstracts references to pieces of data from the actual data itself - and to even store these references in documents separately from a running program. It is possible to replace one object tree with another, but still to maintain a stable reference to the same subproperty `baz`, whoever it happens to be today. This is particularly important in web applications where data claiming to be "your data" can suddenly arrive from anywhere (a JSON feed, some persistence, a particularly aggressive version management system, etc). However it got here, you know it is your data because it is at the right path.
+In some parts of the framework, we refer to "EL expressions". This is a somewhat historical phrase that perhaps sounds like it says more than it is trying to. 
```

Comment:
Given that we support IoC expressions, is it true that our term of "EL expressions" is historical and saying more than it tries to? Or can this comment be removed?